client/src/main/webjar/sidenav/sidenav.html

```diff
@@ -5,4 +5,7 @@
 
     <health-bam-map-query-form></health-bam-map-query-form>
 
+    <health-bam-authentication>
+    </health-bam-authentication>
```

Comment:
Like my other auth0 work, I just threw this in the wrong place (to be cleaned up later).